contents/handbook/engineering/ee-setup.md

```diff
@@ -45,3 +45,14 @@ Currently for internal use only.
   - Tests:
       - All the same, except skip `DEBUG=1` in the env settings.
       - Set as the "target" in run configuration `ee/clickhouse`
+
+### Running everything but ClickHouse locally
```

Comment:
oh I think the section above here works, I think it's just a case of adding `/etc/host` instructions to that?

Reply:
They are sort of different, since now postgres & redis are running locally as well? And ClickHouse is the only thing in docker

Reply:
oh, more importantly, since the above section also assumes plugin server is running inside docker, it doesn't face this specific issue at all.